internal/e2e/rollup/rollup.spec.js

```diff
@@ -16,16 +17,24 @@ if (process.env[update_var] && process.env['TEST_SRCDIR']) {
 }
 
 function check(actual, expected) {
-  if (read(actual) !== read(expected)) {
+  const actualContents = read(actual);
+  const expectedContents = read(expected);
+  if (actualContents !== read(expected)) {
```

Comment:
`read(expected)` should be `expectedContents`

Reply:
Whoops

Reply:
Fixed